src/docs/sdk/performance/opentelemetry.mdx

```diff
@@ -18,7 +18,220 @@ When Sentry performance monitoring was initially introduced, OpenTelemetry was i
+      if (parentSpan) {
+          scope.setSpan(parentSpan);
+      }
+      // We remove the span from the map
+      delete this._map[otelSpanId];
+    }
+  }
+}
+```
+
+Users are required to add this `SentrySpanProcessor` to their OpenTelemetry SDK initialization logic to make this work, like so. Individual SDK implementations might be a little different.
+
+```ts
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { Resource } from "@opentelemetry/resources";
+import * as Sentry from "@sentry/node";
+import { SentrySpanProcessor } from "@sentry/opentelemetry-node";
+
+Sentry.init({
+  /// ...
+});
+
+const sdk = new NodeSDK({
+  resource: new Resource({
+    "service.name": "my-service",
+    "service.version": "1.0.0",
+  }),
+  spanProcessor: new SentrySpanProcessor(),
+});
+```
+
+### Step 2: Define `getTraceData`
+
+We want to make sure a transaction is started with information from the `sentry-trace` and `baggage` headers.
+
+TODO @neel
+
+### Step 3: Define `updateSpanWithOtelData`
+
+We want to update the Sentry Span with the OpenTelemetry data. This includes the span's operation, description, and resource attributes.
+
+The Sentry span description should come from the OpenTelemetry Span name. The Sentry span operation should come from a combinations of the OpenTelemetry Span name, kind, and attributes.
+
+To make things simple, only set ops for `db` and `http` spans. Don't do any other extended logic, this will be done in Relay in the future.
+
+### Step 3: Add OpenTelemetry Context
+
+All SDKs are required to add event context with the key `otel` to events generated by the SpanProcessor. This is detailed in <Link to="./#opentelemetry-context">our spec for the Context</Link>. The OpenTelemetryContext should be generated from information from the OpenTelemetry Span.
+
+```ts
+{
+  contexts: {
+    otel: {
+      attributes: {
+        "http.method": "GET",
+        "http.url": "https://example.com",
+        "http.status_code": 200,
+      },
+      service: {
+        name: "my-service",
+        version: "1.0.0",
+      },
+      otel_sdk: {
+        name: "opentelemetry-node"
+        version: "4.0.0",
+      },
+      // ...
+    }
+  },
+}
+```
+
+### Step 4: Add instrumentor option to Sentry SDK
+
+We want to avoid double instrumenting the same library. To do this, we want to add an option to the Sentry SDK that disables Sentry performance auto-instrumentation when OpenTelemetry is enabled. For now users will have to manually add this option to their SDK initialization code, but in the future we can do this automatically (by detecting if OpenTelemetry exists).
+
+```ts
+Sentry.init({
+  // ...
+  instrumentor: "otel",
+});
+```
+
+You have two options for this:
+
+1. Add an early return to `Sentry.startTransaction` and `Sentry.trace` (a helper method that ) if the `instrumentor` option is set to `otel`. This may require SDK changes, as `trace` is not part of the unified API.
```

Comment:
```suggestion
1. Add an early return to `Sentry.startTransaction()` and `Sentry.trace()` (a helper method) if the `instrumenter` option is set to `otel`. This may require SDK changes, as `Sentry.trace()` is not part of the unified API.